content/en/news/release-notes.md

```diff
@@ -6,6 +6,27 @@ weight: 1
 
 For additional information check our [sprint demo videos](https://www.youtube.com/channel/UCcm2NzDN_UCZKk2yYmOpc5w) and [blogs](https://medium.com/kialiproject).
 
+## 1.49.0
+Sprint Release: April 8th, 2022
+
+Features:
+
+* [Auth: Phase out usage of JWTs](https://github.com/kiali/kiali/issues/4542)
+* [Kiali UI and Kiali Server can point to a single commit](https://github.com/kiali/kiali/issues/4895)
+* [Add cypress UI tests around the Overview page](https://github.com/kiali/kiali/issues/4872)
+* [Update Prometheus client lib](https://github.com/kiali/kiali/issues/4884)
+* [Transfer frontend repo into kiali repo](https://github.com/kiali/kiali/issues/4825)
```

Comment:
is this really a feature?  Do we want this in the release notes?

Reply:
I dunno, could be useful to a community dev.  doesn't hurt.  let's not overthink this :)